src/schedule.rs

```diff
@@ -847,3 +963,45 @@ fn test_no_panic_on_nonexistent_time() {
     let next = schedule.after(&dt).next().unwrap();
     assert!(next > dt); // test is ensuring line above does not panic
 }
+
+#[test]
+fn test_nom_valid_days_of_month_any() {
+    let expression = "* * * ? * *";
```

Comment:
It would also be useful to demonstrate that `?` DOM works with specific DOW, like `"* * * ? * Mon,Thu"`. Same for vice-versa with `?` DOW.